src/icp.cpp

```diff
@@ -122,15 +122,6 @@ usage() {
         "icp copies an irods data-object (file) or collection (directory) to another",
```

Comment:
You can remove the Q in the help text and `optStr` on line 34. Same goes for all the iCommands source files touched in this PR.

Reply:
literal 'Q'

Reply:
Good catch.

Reply:
Did a `git grep Q '*.cpp'`. I think I minded my Qs now, but I still might've missed something, since one letter searches are not the most specific...